src/main/webapp/app/trial/trial.component.ts

```diff
@@ -85,7 +85,8 @@ export class TrialComponent implements OnInit, AfterViewInit {
     }
     importTrials() {
         this.messages = [];
-        this.mongoMessage.content = '';
+        // this.mongoMessage.content = '';
```

Comment:
?

Reply:
This is not used for now but I still want to keep it in case we will use this feature in the future.